legacy-sdk/whirlpool/tests/sdk/whirlpools/client-impl/whirlpool-impl.test.ts

```diff
@@ -1060,15 +1072,9 @@ describe("whirlpool-impl", () => {
           signatures.push(await tx.addSigner(otherWallet).buildAndExecute());
         }
 
-        // To calculate the rewards that have accumulated up to the timing of the close (strictly, decreaseLiquidity),
-        // the block time at transaction execution is used.
-        // TODO: maxSupportedTransactionVersion needs to come from ctx
-        const tx = await ctx.provider.connection.getTransaction(signatures[0], {
-          maxSupportedTransactionVersion: 0,
-        });
-        const closeTimestampInSeconds = new anchor.BN(
-          tx!.blockTime!.toString(),
-        );
+        // To calculate rewards as of close timing under LiteSVM, use the pool's
+        // updated reward_last_updated_timestamp after close.
+        const postClosePoolData = await pool.refreshData();
```

Comment:
is this change a limitation of LiteSVM? if not, i wonder if it makes sense to keep the previous functionality in the spirit of minimizing the test logic diffs